test/commands/endpoint.test.ts

```diff
@@ -114,6 +117,7 @@ describe("endpoint:add", () => {
           foobar: {
             url: "http://foo.baz",
             secret: "fn1234",
+            name: undefined,
```

Comment:
can you update `name` to always be defined? it seems strange to only set it when reading the config, but not when writing it.